requirements.txt

```diff
@@ -17,3 +17,4 @@ tensorflow-text
 tensorboardx
 tensorflow
 tensorboard-plugin-profile
+torch
```

Comment:
This can't go in our dependency closure -- we don't want folks to install torch to dev. It is useful only in this weird situation. Might be worth adding a check to the convert script though?

Reply:
Makes sense. I will add a check to do that.